### ISSUE TITLE
Accessibility audit results Oct 29, 2020

### ISSUE THREAD

**Issue by darthvadur** [all-purpose/ap-website-v3#56]
# Issues

* Live site tested with Lighthouse, IBM Accessibility checker and aXe

## Home and Work pages

- [x] Missing `alt` text for TA Hub image excerpt on home page and work page

## Work page 

- [x] H3 title for case study excerpts should be H2 on this page [bug]

## Case study pages

- [x] H4 heading not necessary in case - needs either article around or just remove heading tag and use the heading class for styling
- [x] `<div class="image-group container" id="">` empty id for this element. (from IBM tool)

## Kumho page

- [x] Missing `alt` text for logo image
- [x] Missing `alt` text for business card image

## TA Hub page

- [x] Missing `alt` text for seven images

## Contact page

- [x] Heading levels are not in sequential order. The "eybrow" styled headings should be H2. The office address can just be the `address` element with appropriate styling
- [x] Underline the "hello@allpurpose.io" to indicate a link. 

## Template

- [x] Set background color for links (nav) in header area. IBM tool fails colour contrast for dark background/light text themes.
